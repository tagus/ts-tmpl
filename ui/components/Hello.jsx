import React from 'react';

const Hello = ({name}) => (
  <h2>Hello there {name}</h2>
)

export default Hello;
