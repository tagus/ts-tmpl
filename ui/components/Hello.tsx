import React from 'react';

const Hello = ({ name }: {name: string}) => (
  <h2 className="font-bold text-xl">
    Hello there <span className="text-blue-600">{name}</span>
  </h2>
);

export default Hello;
