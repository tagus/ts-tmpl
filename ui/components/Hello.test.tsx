import React from 'react';
import Hello from './Hello';
import { shallow } from 'enzyme';

describe('rendering Hello', () => {
  it('renders feed as expected', () => {
    const hello = shallow(<Hello name="Dwight Schrute" />);
    expect(hello).toMatchSnapshot();
  });
});

