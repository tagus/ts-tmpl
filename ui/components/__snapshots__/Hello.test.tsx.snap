// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering Hello renders feed as expected 1`] = `
<h2>
  Hello there 
  Dwight Schrute
</h2>
`;
